src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java

```diff
@@ -211,6 +211,19 @@ public void write(Collection<SinkRecord> records) {
 
       bulkProcessor.add(indexableRecord, flushTimeoutMs);
     }
+
+  }
+
```

Comment:
Let's not introduce this extra blank line unnecessarily. We prefer 1 blank line between methods.
```suggestion
```